src/component/Maps/maps.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./maps.scss";
+import GoogleMapReact from "google-map-react";
+
+import { Icon } from "@iconify/react";
+import locationIcon from "@iconify/icons-mdi/map-marker";
+
+const google_maps_key = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+
+const LocationPin = ({ text }) => (
+  <div className="pin">
+    <Icon icon={locationIcon} className="pin-icon" />
+    <p className="pin-text">{text}</p>
+  </div>
+);
+
+//Map component that takes in a location and zoom level
+const Map = ({ location, zoomLevel }) => {
+  return (
+    <div id="map" style={{ height: "100vh", width: "100%" }}>
```

Comment:
In order to make full usage of React and SCSS, let's add meaningful classNames to these elements. Having an `id` tag is useful when testing, but in terms of readability and usage of the SCSS, we can reference and have cleaner code if we use classNames.